examples/svelte-kit-demo/src/routes/index.svelte

```diff
@@ -2,11 +2,19 @@
 	export const prerender = true;
 </script>
 
+<svelte:head>
+	<title>Home</title>
+</svelte:head>
+
 <h1>Home</h1>
 
-<p>This is an example of using Snowpack for SSR. See <a href="https://gist.github.com/Rich-Harris/3b6f5f7d5dbfb9779a79f77352be6415">this gist</a> for more information</p>
+<p>
```

Comment:
not right now, but we should overhaul this entire example app. it's really only here because it's where SvelteKit started

Reply:
ha. I didn't even notice that this still says Snowpack

should we even keep this example app if we have the others? it seems like the HN and realworld examples demo everything this one does